po/es.po

```diff
@@ -8532,61 +8565,80 @@ msgid ""
 "to [grapheme clusters](https://docs.rs/unicode-segmentation/latest/"
 "unicode_segmentation/struct.Graphemes.html)."
 msgstr ""
+"`String::chars` devuelve un iterador sobre los caracteres reales. Ten en "
+"cuenta que un `char` puede ser diferente de lo que un humano consideraría un "
+"\"carácter\", debido a los [grupos de grafemas](https://docs.rs/unicode-"
```

Comment:
```suggestion
"\"caracter\", debido a los [grupos de grafemas](https://docs.rs/unicode-"
```

"caracter" is not accentuated. This is a common typo. "carácter" refers to a person's behavior (impulsive/calm/etc). "caracter" refers to a character.